pydoctor/test/test_astbuilder.py

```diff
@@ -548,6 +548,47 @@ def method_both():
     assert captured == "mod:14: mod.C.method_both is both classmethod and staticmethod\n"
 
 
+@systemcls_param
+def test_assignment_to_method_in_class(systemcls: Type[model.System]) -> None:
+    """An assignment to a method in a class body does not change the type
+    of the documentable.
+    """
+    mod = fromText('''
+    class Base:
+        def base_method(): ...
+
+    class Sub(Base):
+        base_method = wrap_method(base_method)
+
+        def sub_method(): ...
+        sub_method = wrap_method(sub_method)
+    ''', systemcls=systemcls)
+    assert isinstance(mod.contents['Base'].contents['base_method'], model.Function)
+    assert mod.contents['Sub'].contents.get('base_method') is None
```

Comment:
I don't understand this assertion.... I was expecting that since base_method is redefined in `Sub` it will be part of `Sub`s content.

Also, I was expecting to see an assertion on the docstring content... to check which docstring is used.

Reply:
We ignore the assignment that does the redefinition because it would change the type of `base_method` from `Function` to `Attribute`. That's why `base_method` doesn't exist in the `Sub` class itself.

If there were a `Function` object on the subclass, checking the docstring would be useful. But currently, the only thing we could check is the docstring on the base class, which is not very useful.

Reply:
Maybe also update the docstring with the info that an assigment will just ignore the overwritten method.
I got this. And I think that for now is ok... and later if someone complains we have this documented.

In term of `sub_method` ... I guess that it will have the original docstring and not the docstring from wrap_method...but I think that it is still worth checking it.

Thanks!